GUI/QtQuick/qml/Settings/SettingsGeneral.qml

```diff
@@ -0,0 +1,126 @@
+import QtQuick 2.12
+import QtQuick.Window 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.15
+
+import "../BsStyles"
+import "../StyledControls"
+
+ColumnLayout  {
+
+    id: layout
+
+    signal sig_save()
+
+    height: 548
+    width: 580
+
+    spacing: 0
+
+    CustomTitleLabel {
+        id: title
+
+        Layout.alignment: Qt.AlignCenter
+        Layout.preferredHeight : title.height
+
+        text: qsTr("General")
+    }
+
+    CustomTextInput {
+        id: log_file
+
+        Layout.alignment: Qt.AlignCenter
+        Layout.preferredHeight : 70
+        Layout.preferredWidth: 532
+        Layout.topMargin: 24
+
+        input_topMargin: 35
+        title_leftMargin: 16
+        title_topMargin: 16
+
+        title_text: qsTr("Log file")
+    }
+
+    CustomTextInput {
+        id: messages_log_file
+
+        Layout.alignment: Qt.AlignCenter
+        Layout.preferredHeight : 70
+        Layout.preferredWidth: 532
+        Layout.topMargin: 10
+
+        input_topMargin: 35
+        title_leftMargin: 16
+        title_topMargin: 16
+
+        title_text: qsTr("Messages log file")
+    }
+
+    CustomCheckBox {
+        id: checkbox_advanced_tx
+
+        Layout.alignment: Qt.AlignVCenter | Qt.AlignLeft
+        Layout.topMargin: 24
+        Layout.leftMargin: 24
+
+        text: "Advanced TX dialog by default"
+
+        spacing: 6
+        font.pixelSize: 13
+        font.family: "Roboto"
+        font.weight: Font.Normal
+
+        checked: true
+    }
+
+    Label {
+        id: spacer
+        Layout.fillWidth: true
+        Layout.fillHeight: true
+    }
+
+    CustomButton {
+        id: save_but
+
+        Layout.bottomMargin: 40
+        Layout.alignment: Qt.AlignBottom | Qt.AlignHCenter
+
+        enabled: log_file.input_text.length && messages_log_file.input_text.length
+
+        width: 532
+
+        text: qsTr("Save")
```

Comment:
We don't have "Save" button in mockup. It's only for adding a new row into the Armory servers model (to be added later). Each setting should be saved by assigning to a property in corresponding control.

Reply:
We have mockup with other functionaluty))) (I mean figma) But ok I will remove Save button

Reply:
If we will change 1) port of armory (immediantly set data in c++) then 2) host of armory (immediantly set data in c++) . Between 1) and 2) step we will have wrong armory configuration - can be it problem or not?

Reply:
No, it won't harm now, program restart will be required for this setting to take effect (together with testnet/environment change).